content/api/ip-pools.apib

```diff
@@ -6,19 +6,50 @@ description: Manage IP pools, which allow you to group and assign dedicated IPs
 
 <banner status="info"><strong>Note</strong>: This endpoint is available for SparkPost customers only. <a href="https://www.sparkpost.com/enterprise-email/">SparkPost Enterprise</a> customers, contact your TAM for details.</banner>
 
-#### IP Pool Properties
+## IP Pool Object
 
-| Property   | Type    | Description | Notes |
-|------------|---------|-------------|-------|
-| id | string | Unique pool identifier | Autogenerated on create, used for Transmissions API & SMTP |
-| name | string | Pool name | |
-| signing_domain | string | DKIM verified signing domain | |
-| fbl_signing_domain  | string            | FBL Signing Domain for the pool | Used only for emails to domains that require DKIM for FBL    |
-| ips | array of objects | Sending IPs within the pool | Sending IP objects include fields `external_ip` & `hostname` |
++ Data Structure (sample)
```

Comment:
indentation might be wrong, not rendering correctly for me

Reply:
nah, it is that it says `(sample)` which got removed when we moved to the `+ Sample` syntax